backend/app.py

```diff
@@ -14,41 +14,35 @@
 CACHE_CONFIG = {"CACHE_TYPE": "filesystem", "CACHE_DIR": "/tmp"}
 cache = Cache(app, config=CACHE_CONFIG)
 
-AIRTABLE_API = "https://api.airtable.com/v0/"
-AIRTABLE_BASE = "appISQqfvRPMg6CH3/Suivi"
-API_KEY = os.getenv("AIRTABLE_API_KEY")
-HEADERS = {"Authorization": f"Bearer {API_KEY}"}
+NOTION_API = "https://api.notion.com/v1/databases/"
+NOTION_BASE = "323a4e5283e54dd28c616e34da86b3d9"
```

Comment:
I don't think I could access this notion base? Should [we use this one](https://www.notion.so/apigouv/19dfba28f5cc4f2688fab34d383283f6) @agarrone for testing?

In any case, we could make it as an env if we want to have a preprod? :)

Reply:
I think thats the right base yes

Reply:
I'll change it to an env value. This is a copy of this base in a private workspace where I'm admin. We can use the one from the open data cockpit when a Notion integration is created by an admin. I'm documenting this in the parent issue : https://github.com/etalab/data.gouv.fr/issues/1057

Reply:
There is a `NOTION_BASE` env variable now.